### DISCUSSION TITLE
Ant Design Vue table 自定义展示添加方法后报错

### DISCUSSION THREAD

**Discussion by lichao1233** [opentiny/tiny-engine#946] [Q&A]
![企业微信20241219-185730](https://github.com/user-attachments/assets/f7c2dbda-c24d-4bc4-9eae-96577f80fb8a)

**Comment by lichao1233** [comment]
这是配置json
![企业微信20241219-190040](https://github.com/user-attachments/assets/d2928260-f88e-4db9-9f3e-ff1dfa73d5eb)

**Comment by chilingling** [comment]
@lichao1233  从截图来看，出码不正确，方便提供下生成的 schema 嘛？
